go-controller/pkg/cluster/management_port.go

```diff
@@ -250,23 +255,56 @@ func CreateManagementPort(nodeName, localSubnet string, clusterSubnet []string)
 		}
 	}
 
+	// Set macAddress as annotation
+	var node *kapi.Node
+	node, err = kube.GetNode(nodeName)
+	if err != nil {
+		logrus.Errorf("Error nodeName %s, notfound - %v", nodeName, err)
+		return err
+	}
+	err = kube.SetAnnotationOnNode(node, OvnNodeManagementPortMacAddress, macAddress)
+	if err != nil {
+		logrus.Errorf("Failed to set node %s macAddress annotation: %v", nodeName, macAddress)
+		return err
+	}
+
+	portName := "k8s-" + nodeName
+
+	// Wait for the portMac to be created
+	if err := wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
+		portMac, portIP, err := util.GetPortAddresses(portName)
+		if err != nil {
+			logrus.Errorf("Error while obtaining addresses for %s - %v", portName,
+				err)
+			return false, err
+		}
+		if portMac == nil || portIP == nil {
+			return false, nil
+		}
+		return true, nil
+	}); err != nil {
+		logrus.Errorf("Timeout error while obtaining addresses for %s "+
+			"stdout: %q, stderr: %q, (%v)", portName, stdout, stderr, err)
+		return err
+	}
+
 	// Create this node's management logical port on the node switch
```

Comment:
@mccv1r0 we should change this comment, it's no longer accurate since the master does this now.